### DISCUSSION TITLE
I was told that I could not apply for verification as a student

### DISCUSSION THREAD

**Discussion by pilihualabi** [community/community#86474] [GitHub Education]
### Hi everyone,

In March, 23, I was told that I needed to be verified as a student again. After submitting the information again, I was informed by email in April, 23, that I could not apply for verification as a student. the email mentioned "Your account is no longer able to apply after we found indications it was used to fraudulently obtain the benefits  offered as part of the Student Developer Pack.
Further attempts may result in permanent account suspension. ".
This makes me feel confused, I have never given my account to anyone else, and I am indeed a college student now, and I would like an explanation and re-verification of my student status.

**Comment by knownothing10** [comment]
In this situation, the best course of action would be to reach out to GitHub support directly. They will be able to review your account and provide you with an explanation of the situation.

Contact GitHub support by visiting the GitHub Support website, navigating to the appropriate section, and submitting a support ticket or reaching out to them through their official channels. Explain the issue, mention that you are a college student, and request an explanation and re-verification of your student status.

GitHub support will be able to provide you with more accurate information and guide you through the process of resolving this issue.

**Reply by pilihualabi** [reply]
Thanks for your advice, I actually tried to ask GitHub Support for help before, but the problem went nowhere, I thought maybe I should try again
